sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_patch.py

```diff
@@ -140,8 +140,8 @@ class JwtCredentialPolicy(SansIOHTTPPolicy):
 
     NAME_CLAIM_TYPE = "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name"
 
-    def __init__(self, credential, user=None):
-        # type: (AzureKeyCredential, typing.Optional[str]) -> None
+    def __init__(self, credential, **kwargs):
```

Comment:
This is a breaking change (we are removing the possibility to pass in the `user` as a positional parameter....

Reply:
JwtCredentialPolicy isn't exposed to users, i feel like that means that it's technically breaking, but we only break ourselves. I might be missing something though

Reply:
Oh, I thought this was a public class. Did we make it internal?

Reply:
yeah i don't think we ever exposed it. Not any code in the [init](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/__init__.py), and i'm not seeing it in [apiview](https://apiview.dev/Assemblies/Review/f23ba872c2474933ad39eb00652bcfdf)